parquet-format-structures/src/test/java/org/apache/parquet/format/TestLogicalTypes.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.format;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class TestLogicalTypes {
+  @Test
+  public void testVariantLogicalTypeVersion() {
+    LogicalType variant = LogicalTypes.VARIANT();
+    assertEquals(1, (variant.getVARIANT().getSpecification_version()));
```

Comment:
I don't really see the value of this test. This is testing that a constant has some expected value.

Reply:
I was trying to make sure the version is as expected, but otherwise, it doesn't do much. Let me remove for now.